source/module_elecstate/elecstate_lcao_tddft.cpp

```diff
@@ -67,7 +67,9 @@ void ElecStateLCAO_TDDFT::psiToRho_td(const psi::Psi<std::complex<double>>& psi)
     Gint_inout inout(this->loc->DM_R, this->charge->rho, Gint_Tools::job_type::rho); // rho calculation
     this->uhm->GK.cal_gint(&inout);
 
+    std::cout << "this->charge->renormalize_rho(); 1" << std::endl; 
```

Comment:
this sentence should be deleted? otherwise unexpected information will be printed out?

Reply:
done